config/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ if(IMATH_INSTALL_PKG_CONFIG)
   endfunction()
   imath_pkg_config_help(../Imath.pc.in)
 else()
-  message(NOTICE "-- WARNING pkg-config generation disabled")
+  message(STATUS "-- WARNING pkg-config generation disabled")
```

Comment:
There is also "WARNING", might it be appropriate here?

Reply:
Also, I think that if you use STATUS, you can remove the "-- " because that's automatically added by status.

Reply:
The only way to get here is by explicitly setting IMATH_INSTALL_PKG_CONFIG=0, so this message is simply confirming what settings have been provided.  Warning implies there might be something wrong, which doesn't seem appropriate. Status seems appropriate since it's merely echoing what it's been told.

Reply:
Then why does the message itself say "WARNING"?

Reply:
N/M, I see you removed that word now